### DISCUSSION TITLE
Tropical Cyclone Statement not listed under /alerts/types

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#296] [Q&A]
Very minor issue...

"Tropical Cyclone Statement" is a common alert event currently. However, it is not listed under api.weather.gov/alerts/types. I'm assuming this is just an oversight (but correct me if it's something else).

FWIW it isn't listed on this page either: https://www.weather.gov/help-map

**Comment by qconrad** [comment]
I noticed these show up on weather.gov as "Hurricane Local Statement"s. Not sure if that is why.
